### ISSUE TITLE
Number of columns in index on load

### ISSUE THREAD

**Issue by fabianmoronzirfas** [basiljs/basiljs.github.io#34]
in my system I'm getting fice columns on the first load of the site. After I resize it changes to three.
![cleanshot 2018-07-31 at 19 37 38](https://user-images.githubusercontent.com/315106/43476749-9f473e44-94f9-11e8-8e51-fa61c006bb0c.png)

**Comment by ffd8** [comment]
That's true - saw this the other week with Kaj and he's looking into it.

**Comment by fabianmoronzirfas** [comment]
This was based on me having to init the column creation on the reference template. Which my plays into the point for issue #79 

Actually the JQuery plugin together with my data model is also a mess. I guess we should rebuild the column layout with CSS Grid and remove JQuery as a dependency for that.
